eclass/kernel-build.eclass

```diff
@@ -656,12 +655,22 @@ kernel-build_merge_configs() {
 
 	# Only semi-related but let's use that to avoid changing stable ebuilds.
 	if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
```

Comment:
BTW, can't this condition go now? (in a followup)

Reply:
> BTW, can't this condition go now? (in a followup)

Not yet, we still have the 5.10, 5.15 and 6.1 series in the tree, for which we have not implemented `USE=modules-compress` and `USE=generic-uki`. They are guarded behind the same variable for historical/convenience reasons.

Reply:
Ah, thanks.